src/core/Microsoft.PowerToys.Settings.UI.Lib/ViewModels/GeneralViewModel.cs

```diff
@@ -36,7 +36,7 @@ public class GeneralViewModel : Observable
         public GeneralViewModel(ISettingsRepository<GeneralSettings> settingsRepository, string runAsAdminText, string runAsUserText, bool isElevated, bool isAdmin, Func<string, int> updateTheme, Func<string, int> ipcMSGCallBackFunc, Func<string, int> ipcMSGRestartAsAdminMSGCallBackFunc, Func<string, int> ipcMSGCheckForUpdatesCallBackFunc, string configFileSubfolder = "")
         {
             CheckFoUpdatesEventHandler = new ButtonClickCommand(CheckForUpdates_Click);
```

Comment:
nit: CheckFoUpdatesEventHandler  "CheckFo" should be "Check**For**"  note the missing R